src/apps/loan-list/list/loan-list.dev.tsx

```diff
@@ -9,6 +9,10 @@ export default {
   title: "Apps / Loan list",
   component: LoanList,
   argTypes: {
+    materialOverdueUrl: {
+      defaultValue: "https://unsplash.com/photos/wd6YQy0PJt8", // open source image of a red panda
```

Comment:
"open source image of a red ANGRY panda" would be more precise

Reply:
I thought it was yawning. Which is adorable.